src/ansys/fluent/core/services/field_data.py

```diff
@@ -942,8 +1091,9 @@ def get_scalar_field_data(
 
         Returns
         -------
-        Dict[int, np.array]
-            Dictionary containing a map of surface IDs to the scalar field.
+        Union[ScalarFieldData, Dict[int, ScalarFieldData]]
+            ScalarFieldData if surface name is provided as input.
+            Dictionary containing a map of surface IDs to ScalarFieldData if surface_ids are provided as input.
```

Comment:
```suggestion
            If a surface name is provided as input, scalar field data is returned. If surface
            IDs are provided as input, a dictionary containing a map of surface IDs to scalar
            field data.
```
The data type shows the object names, allowing the description to be in plain English.